pkgs/top-level/all-packages.nix

```diff
@@ -13467,21 +13467,16 @@ with pkgs;
     inherit (darwin.apple_sdk.libs) Xplugin;
   };
 
-  xorg = recurseIntoAttrs (lib.callPackagesWith pkgs ../servers/x11/xorg {
-    inherit clangStdenv fetchurl fetchgit fetchpatch stdenv intltool freetype fontconfig
-      libxslt expat libpng zlib perl mesa_drivers spice-protocol libunwind
-      dbus libuuid openssl gperf m4 libevdev tradcpp libinput mcpp makeWrapper autoreconfHook
-      autoconf automake libtool mtdev pixman libGL libGLU
-      cairo epoxy;
-    inherit (buildPackages) pkgconfig xmlto asciidoc flex bison;
-    inherit (darwin) apple_sdk cf-private libobjc;
+  xorg = recurseIntoAttrs ((lib.callPackagesWith pkgs ../servers/x11/xorg {
+  }).overrideScope (lib.callPackageWith pkgs ../servers/x11/xorg/overrides.nix {
```

Comment:
Can we do inline https://github.com/NixOS/nixpkgs/blob/f305b9b0862f28a85a3dcc2a517d4fcb345abc42/lib/customisation.nix#L201 to get something like
```
(lib.fixedPoints.extends (import ../servers/x11/xorg/overrides.nix) (import ../servers/x11/xorg)
```
(This ignores the flip. That would be for the odd order of `self: super:`, when you already did idiomatic `self: super:`.)